CHANGES.md

```diff
@@ -1,3 +1,23 @@
+Release 4.39 (release date: 2019-05-29)
+=======================================
+New Features
+------------------
+- Compression level is now configurable by setting the `gzip_compression_level`
+  option under `[GSUtil]`. The default value is `9`.
```

Comment:
Other spots in the CHANGES.md file use this phrasing/notation:

`... setting "gzip_compression_level" under the "GSUtil" section in the boto config file.`

Also, I'd advise against putting the 9 in any sort of ticks/quotes.  Boto files are very picky about string vs integer values, and someone will inevitably mistake those for single quotes and think the value should be a quoted string :upside_down_face: